app/models/project.rb

```diff
@@ -170,7 +170,7 @@ def set_last_synced_at
   end
 
   def async_sync
-    PackageManagerDownloadWorker.perform_async(platform, name)
+    PackageManagerDownloadWorker.perform_async(platform_class, name)
```

Comment:
The way this is written, we will not be able to correctly sync something not on Maven Central. This eventually ends up at `PackageManager::Maven.update()` so maybe we can do something in there to try and figure out the right provider to use.